compiler/rustc_middle/src/query/mod.rs

```diff
@@ -8,6 +8,7 @@ use crate::ty::{self, print::describe_as_module, TyCtxt};
 use rustc_span::def_id::LOCAL_CRATE;
 
 mod keys;
+pub mod on_disk_cache;
```

Comment:
This module should stay inside `rustc_query_impl`.

Reply:
I would like to split the module up in future PRs to avoid growing this one too large.